src/layerInfos/GeltungsbereicheLayerInfo/GeltungsbereicheLayerInfo.js

```diff
@@ -34,8 +34,71 @@ const infos = {
   },
 };
 
+const warranty = {
+  de: (
+    <div>
+      Alle Angaben ohne Gewähr. Änderungen vorbehalten.
+      <br />
+      Es gilt der Anwendungsbereich (
+      <a
+        href="https://www.allianceswisspass.ch/de/tarife-vorschriften/uebersicht"
+        rel="noopener noreferrer"
+        target="_blank"
+      >
+        Übersicht der Tarife und Vorschriften - Alliance SwissPass
+      </a>
+      ).
+    </div>
+  ),
+  fr: (
+    <div>
+      Toutes les indications sont sans garantie. Sous réserve de modifications.
+      <br />
+      Le champ d&apos;application fait foi (
+      <a
+        href="https://www.allianceswisspass.ch/fr/tarifs/TarifsPrescriptions"
+        rel="noopener noreferrer"
+        target="_blank"
+      >
+        Tarifs et prescriptions Vue d&apos;ensemble - Alliance SwissPass
+      </a>
+      ).
+    </div>
+  ),
+  en: (
+    <div>
+      All information without guarantee. Subject to change.
+      <br />
+      The scope of application applies (
+      <a
+        href="https://www.allianceswisspass.ch/de/tarife-vorschriften/uebersicht"
+        rel="noopener noreferrer"
+        target="_blank"
+      >
+        Übersicht der Tarife und Vorschriften - Alliance SwissPass german only
```

Comment:
The "german only" should be outside the <a>-tag

Reply:
done